app/src/main/java/com/gh4a/model/StatusWrapper.java

```diff
@@ -95,7 +99,11 @@ public String targetUrl() {
         return mTargetUrl;
     }
 
-    private static String formatTimeDelta(Context context, Date start, Date end) {
+    @Nullable
+    private static String formatTimeDelta(Context context, @Nullable Date start, @Nullable Date end) {
+        if (start == null || end == null)
+            return null;
```

Comment:
Please add curly braces here.

Reply:
Done!

Reply:
Great, now just please squash both commits into one and I'll merge :-)

Reply:
I can't do that at the moment, but you can "Squash and merge" from GitHub UI which does the same thing :D

Reply:
I can even do that from Octodroid ;-) but I don't want merge commits in the history.